lib/rubocop/cop/rspec/capybara/specific_matcher.rb

```diff
@@ -28,6 +28,28 @@ class SpecificMatcher < Base
           MSG = 'Prefer `%<good_matcher>s` over `%<bad_matcher>s`.'
           RESTRICT_ON_SEND = %i[have_selector have_no_selector have_css
                                 have_no_css].freeze
+          HAVE_BUTTON_OPTIONS = %w[
```

Comment:
Perhaps we can extract `COMMON_OPTIONS` and then extend for the different elements

Reply:
It certainly looks better that way. Thank you very much.